src/seedu/addressbook/parser/Parser.java

```diff
@@ -75,33 +63,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
 
-        case AddCommand.COMMAND_WORD:
-            return prepareAdd(arguments);
+            case AddCommand.COMMAND_WORD:
```

Comment:
Based on our coding standard, `case` should have the same indentation as `switch` statements.